pre_commit/resources/empty_template_go.mod

```diff
@@ -0,0 +1 @@
+module github.com/pre-commit/pre-commit-dummy
```

Comment:
will this cause it to try and install this module? can this file be empty?

Reply:
I don't think it causes any fetching. Haven't tested with an empty one, will do so now.

Reply:
Yeah, empty one appears to work just fine as well, tested with 1.13 and 1.14.

Found this info nugget about empty go.mod: https://github.com/golang/go/wiki/Modules#can-an-additional-gomod-exclude-unnecessary-content-do-modules-have-the-equivalent-of-a-gitignore-file

Would you prefer it empty?

Reply:
yeah empty seems good

Reply:
Emptied, force-pushed.